qa/rpc-tests/rest.py

```diff
@@ -151,11 +143,11 @@ def run_test(self):
         binaryRequest += struct.pack("i", 0);
 
         bin_response = http_post_call(url.hostname, url.port, '/rest/getutxos'+self.FORMAT_SEPARATOR+'bin', binaryRequest)
-        output = StringIO.StringIO()
+        output = io.BytesIO()
         output.write(bin_response)
         output.seek(0)
         chainHeight = struct.unpack("i", output.read(4))[0]
-        hashFromBinResponse = hex(deser_uint256(output))[2:].zfill(65).rstrip("L")
```

Comment:
It seems like I might have introduced a bug here.   Is it that in the old version there was an "L" on the end and that's why the argument to `zfill` had to be one greater?  I'm not sure.   I need to check into this before the code is merged.